weaviate/gql/get.py

```diff
@@ -610,6 +619,546 @@ def with_near_image(self, content: dict, encode: bool = True) -> "GetBuilder":
         self._contains_filter = True
         return self
 
+    def with_near_audio(self, content: dict, encode: bool = True) -> "GetBuilder":
+        """
+        Set `nearAudio` filter.
+
+        Parameters
+        ----------
+        content : dict
+            The content of the `nearObject` filter to set. See examples below.
+        encode : bool, optional
+            Whether to encode the `content["audio"]` to base64 and convert to string. If True, the
+            `content["audio"]` can be an audio file path or a file opened in binary read mode. If False,
+            the `content["audio"]` MUST be a base64 encoded string (NOT bytes, i.e. NOT binary
+            string that looks like this: b'BASE64ENCODED' but simple 'BASE64ENCODED').
+            By default True.
+
+        Examples
+        --------
+        Content prototype:
+
+        >>> content = {
+        ...     'audio': <str or binary read file>,
+        ...     # certainty ONLY with `cosine` distance specified in the schema
+        ...     'certainty': <float>, # Optional, either 'certainty' OR 'distance'
+        ...     'distance': <float>, # Optional, either 'certainty' OR 'distance'
+        ... }
+
+        >>> {
+        ...     'audio': "e5dc4a4c-ef0f-3aed-89a3-a73435c6bbcf",
+        ...     'certainty': 0.7 # or 'distance'
+        ... }
+
+        With `encoded` True:
+
+        >>> content = {
+        ...     'audio': "my_audio_path.png",
+        ...     'certainty': 0.7 # or 'distance' instead
+        ... }
+        >>> query = client.query.get('Image', 'description')\\
+        ...     .with_near_audio(content, encode=True) # <- encode MUST be set to True
+
+        OR
+
+        >>> my_audio_file = open("my_audio_path.png", "br")
```

Comment:
the examples are sometimes not consistent with the function, for example here is a .png file used as audio file

Reply:
fixed